### PR TITLE
Feature Management: add documentation

### PR DESCRIPTION
**Pull request by jcalisto**
**What is this feature?**

Adding documentation for the new Feature Management page, to be released soon. 

Related with https://github.com/grafana/grafana-enterprise/issues/4897

### CODE REVIEW

File: docs/sources/administration/feature-management/index.md
Comment (jdbaldry) [inline docs/sources/administration/feature-management/index.md:31]: There is a shortcode for consistent admonitions: https://grafana.com/docs/writers-toolkit/write/shortcodes/#admonition

Prefer U.S. English (behavior, not behaviour). 

```suggestion
{{< admonition type="note" >}}
Options marked with an asterisk (\*) are defaults for the corresponding feature stage.
Each feature toggle owner can override its default behavior.
{{< /admonition >}}
```

File: docs/sources/administration/feature-management/index.md
Comment (jdbaldry) [inline docs/sources/administration/feature-management/index.md:33]: Sentence casing

```suggestion
## Edit feature toggles
```

File: docs/sources/administration/feature-management/index.md
Comment (imatwawana) [inline docs/sources/administration/feature-management/index.md:22]: Using bold formatting for UI elements/pages
In the last sentence I changed the wording because it sounds like something we're going to do in the future vs something that already exists.
```suggestion
In the **Administration** page, the **Feature Management** section enables administrators to see and switch the various features that are active in their Grafana environment. Based on the development stage of the feature, there are specific options for visibility and customization.
```

File: docs/sources/administration/feature-management/index.md
Comment (imatwawana) [inline docs/sources/administration/feature-management/index.md:29]:
```suggestion
| Private preview       | Visible\*  | No       |
| Public preview       | Visible\*  | No       |
| General availability | Visible\*  | Yes\*    |
```

File: docs/sources/administration/feature-management/index.md
Comment (jdbaldry) [inline docs/sources/administration/feature-management/index.md:35]: Prefer "refer to" over "see".


```suggestion
You can only edit feature toggles if Grafana is configured with the proper feature management settings.
To configure feature management, refer to [Configure feature management](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/feature-toggles/#configure-feature-management).
```

File: docs/sources/administration/feature-management/index.md
Comment (imatwawana) [inline docs/sources/administration/feature-management/index.md:31]: Using admonition shortcode
```suggestion
{{% admonition type="note" %}}
Options marked with an asterisk (\*) are defaults for the corresponding feature stage. Each feature toggle owner can override its default behaviour.
{{% /admonition %}}
```

File: docs/sources/administration/feature-management/index.md
Comment (jdbaldry) [inline docs/sources/administration/feature-management/index.md:44]:
```suggestion
1. Sign in to Grafana as a server administrator.
1. Click **Administration** in the left-side menu, **General**, and then **Feature Toggles**.
1. Navigate the list of feature toggles and select your feature state overrides.
1. Click **Save changes** and wait for your Grafana instance to restart with the updated feature toggles.
```

Our Markdown renderer will autoincrement the steps.

File: docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md
Comment (jdbaldry) [inline docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md:180]: Sentence casing

```suggestion
## Configure feature management
```

File: docs/sources/administration/feature-management/index.md
Comment (imatwawana) [inline docs/sources/administration/feature-management/index.md:35]: Changing this to active voice and adding a version syntax parameter so the system can always choose the appropriate version.
```suggestion
You can only edit feature toggles if Grafana is configured with the proper feature management settings. For more information, refer to [Configure Feature Management](/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/feature-toggles/#configure-feature-management").
```

File: docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md
Comment (jdbaldry) [inline docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md:182]:
```suggestion
**Feature Management** is an Administration page that allows admin users to visualize and toggle the feature flags available in their Grafana instance, for more information, refer to [Feature Management](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/administration/feature-management/).
```

File: docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md
Comment (jdbaldry) [inline docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md:184]:
```suggestion
By default, feature toggles are in read-only mode.
Granting admin users the ability to alter the states of feature toggles requires configuring Grafana with the optional [allow_editing](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#allow_editing), [update_webhook](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#update_webhook) and [update_webhook_token](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#update_webhook_token) settings.
```

File: docs/sources/administration/feature-management/index.md
Comment (imatwawana) [inline docs/sources/administration/feature-management/index.md:39]:
```suggestion
To edit a feature toggle, follow these steps:
```

File: docs/sources/administration/feature-management/index.md
Comment (imatwawana) [inline docs/sources/administration/feature-management/index.md:42]: Confirm capitalization of "Feature toggles" in the UI and follow that.

File: docs/sources/administration/feature-management/index.md
Comment (imatwawana) [inline docs/sources/administration/feature-management/index.md:43]:
```suggestion
3. Navigate to the list of feature toggles and select your feature state overrides.
```

**Review by jdbaldry — COMMENTED** [review]
Thanks for the contribution!

I think having a technical writer review the prose makes sense because I'm not one myself.

I have provided some suggestions to use some best practices documented in [Writers' Toolkit](https://grafana.com/docs/writers-toolkit/).

nit: 
Consider using semantic line breaks.
[Semantic line breaks](https://sembr.org/) are better for future line based diffing in the GitHub UI and result in the same HTML output.

File: docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md
Comment (imatwawana) [inline docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md:182]:
```suggestion
**Feature management** is an **Administration** page that allows admin users to visualize and toggle the feature flags available in their Grafana instance. For more information, refer to [Feature Management](/docs/grafana/<GRAFANA_VERSION>/administration/feature-management/).
```

File: docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md
Comment (imatwawana) [inline docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md:184]:
```suggestion
By default, feature toggles are in read-only mode. Granting admin users the ability to alter the states of feature toggles requires configuring Grafana with the optional [allow_editing](/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana#allow_editing), [update_webhook](/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana#update_webhook), and [update_webhook_token](/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana#update_webhook_token) settings.
```

**Review by imatwawana — COMMENTED** [review]
Thanks for the contribution! I've made a number of style edits and left a couple questions in places where we should be matching the UI in the way we write the documentation. If you want me to take another look once you've made changes, just ping me in the comments or re-request review.

File: docs/sources/administration/feature-management/index.md
Comment (jcalisto) [reply docs/sources/administration/feature-management/index.md]: Updated the title to Feature toggles as per Jack's suggestion, we refer to the **Feature Toggles** UI page below.

**Comment by jcalisto** [comment]
Thanks @jdbaldry and @imatwawana for the suggestions! I included your feedback, feel free to have a second look

File: docs/sources/administration/feature-toggles/index.md
Comment (mmandrus) [reply docs/sources/administration/feature-toggles/index.md:2]: nit, but technically admin is not required

Comment (mmandrus) [reply docs/sources/administration/feature-toggles/index.md:2]: consider changing uses of the word "admin" throughout the article too to reflect this

File: docs/sources/administration/feature-toggles/index.md
Comment (mmandrus) [inline docs/sources/administration/feature-toggles/index.md:30]:
```suggestion
| Deprecated             | Visible\*  | Yes\*    |

```

File: docs/sources/administration/feature-toggles/index.md
Comment (mmandrus) [inline docs/sources/administration/feature-toggles/index.md:40]: This seems like a good place to mention that this is supported in all tiers of Grafana Cloud.

**Review by mmandrus — CHANGES_REQUESTED** [review]
Looking good! A few comments.

**Review by mmandrus — APPROVED** [review]

**Comment by imatwawana** [comment]
@jcalisto - This document changed pretty dramatically from the last time I reviewed it and I'm not sure everything that was removed should have been taken out, but I didn't have a chance to review it again before it was merged (I wanted to wait until Michael finished his review). I'll take a look and open a new PR if I think it's necessary.

**Pull request merged**
